src/extension.ts

```diff
@@ -134,6 +134,18 @@ export function activate(context: vscode.ExtensionContext) {
 	registerCommand('cosmosDB.api.getDatabase', async () => {
 		return (<IAzureParentNode>await tree.showNodePicker([MongoDatabaseTreeItem.contextValue, DocDBDatabaseTreeItem.contextValue])).id;
 	});
+	registerCommand('cosmosDB.api.getConnectionString', async (treeItemId: string) => {
+		const node = (<IAzureNode | undefined>await tree.findNode(treeItemId));
```

Comment:
nit: Casting to `IAzureNode | undefined` should be unnecessary. That should already be the return type of findNode